mbuild/tests/test_compound.py

```diff
@@ -1060,10 +1060,12 @@ def test_from_pybel_residues(self):
        cmpd.from_pybel(pybel_mol)
        assert 'LIG1' in cmpd.children[0].name
 
+    @pytest.mark.parametrize('extension', ['pdb', 'sdf'])
     @pytest.mark.skipif(not has_openbabel, reason="Pybel is not installed")
```

Comment:
we use parmed directly to convert to pdb right? If we dont have pybel installed, this test will skip and miss checking for pdb.

Reply:
Actually, i might be wrong. It might skip only the sdf, since the pdb will be its own test due to the parametrize mark

Reply:
Both of the tests from `test_from_pybel_molecule` (for sdf and pdb) will get skipped if pybel is not installed.  Not sure if this answers your question.

Reply:
Partially, isnt pdb handled by parmed? I guess my question is, do we need to test for this, since we use parmed for our pdb? Or is pybel the one we want to use for pdbs?

Reply:
I understand now, that seems perfectly fine the way you have it set up then!